CONTRIBUTING.md

```diff
@@ -31,63 +31,53 @@ More specifically:
+# UV Package Manager
 
-Possible types are:
-
-- `feature`: Signifying a new feature.
-- `bugfix`: Signifying a bugfix.
-- `documentation`: Signifying a documentation improvement.
-- `removal`: Signifying a deprecation or removal of public API.
-
-For changes that do not fall under any of the above cases, please specify the lack of the changelog in the pull request
-description so that a maintainer can skip the job that checks for newly added fragments.
-
-Best way to create the fragments is to run `towncrier create {pull_request_number}.{type}.md` after creating the
-pull request, edit the created file and committing the changes.
+We use the `uv` package manager as a faster drop-in replacement for pip. While it's not required to use in order to
+contribute to hikari, it's highly recommended! From here on we will use `uv` instead of `pip`.
 
-Multiple fragment types can be created per pull request if it covers multiple areas.
+If you still want to use `pip` as your package manager, you can take a look at equivalent `pip`command by clicking on
+the dropdowns below every command.
 
-# Branches
+Installing `uv` is explained [here](https://docs.astral.sh/uv/getting-started/installation/)
 
-We would like to keep consistency in naming branches in the remote.
+# Getting Started
 
-To push branches directly to the remote, you will have to name them like this:
+First of all you should start by cloning the repository.
+In the repository, make a virtual environment 
+```bash
+uv venv
+```
+<details>
+    <summary>Equivalent pip command</summary>
+    
+```bash
+python -m venv .venv
+```
+</details>
 
-- `feature/issue-number-small-info-on-branch`
-    - This should be used for branches that require more tasks to merge into before going as one MR into `master`.
-- `bugfix/issue-number-small-info-on-branch`
-    - This should be used for bugfixes.
-- `task/issue-number-small-info-on-branch`
-    - This should be the default for any commit that doesn't fall in any of the cases above.
+and enter it (`source .venv/bin/activate` on
+Linux, or for Windows use one of `.venv\Scripts\activate.ps1`, `.venv\Scripts\activate.bat`,
+`source .venv/Scripts/activate`).
 
-`issue-number` is optional (only use if issue exists) and can be left out. `small-info-on-branch` should be replaced
-with a small description of the branch.
+Then you should continue by installing [nox](#nox)
 
 # Nox
 
 We have nox to help out with running pipelines locally and provides some helpful functionality.
 
 You will need to install `nox` locally before running any pipelines.
 
-> [!TIP]
-> We use the `uv` package manager as a faster drop-in replacement for pip. While it's not required to use in order to
-> contribute to hikari, it's highly recommended! If you use it, replace `pip` commands below with `uv pip`.
-
 ```bash
-pip install --group nox
+uv sync --group nox
+```
+<details>
+    <summary>Equivalent pip command</summary>
+    
+```bash
+pip install nox[uv]
```

Comment:
```suggestion
pip install 'nox[uv]'
```
Quoted it because some commonly used shells on *nix environment don't tolerate `[square brackets]` the way you'd expect. This should be supported universally (I hope)